src/components/dashboard/InventoryModal.tsx

```
@@ -0,0 +1,194 @@
+    } finally {
+      setIsDownloading(false);
+      // setDownloadProgress(0);
+      onClose();
+    }
+  };
+  return (
+    <div
+      ref={modalRef}
+      className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50"
+      onClick={handleClickOutside}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)